addons/website_event_track_exhibitor/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_chat_room_manager,chat.room.manager,website_jitsi.model_chat_room,event.group_event_manager,1,1,1,1
```

Comment:
Is it new ?

Reply:
@tde-banana-odoo I think we remove it and after my rebase it recreate the file. I remove it and re-push